client/src/main/java/org/eclipse/hono/client/DelegatedCommandSender.java

```diff
@@ -0,0 +1,21 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+
+package org.eclipse.hono.client;
+
+/**
+ * A sender to send delegated command messages.
```

Comment:
That is a command message, first retrieved from the *tenant-scoped* consumer in the protocol adapter, and then *delegated* back to the AMQP message network so that it can be consumed by the *device-specific* consumer later.
I can extend the javadoc there.

Reply:
See [commit](https://github.com/eclipse/hono/compare/08dfcff597695d2a40c589f4d25a36d52a5d470a..dcca29de3f945fbec1b912f5d6626174c4b5f341) below.